src/azure-cli/setup.py

```diff
@@ -76,7 +76,7 @@
     'azure-mgmt-compute~=21.0.0',
     'azure-mgmt-consumption~=2.0',
     'azure-mgmt-containerinstance~=1.4',
-    'azure-mgmt-containerregistry==3.0.0rc17',
+    'azure-mgmt-containerregistry==8.0.0',
```

Comment:
For when we're releasing a new feature we build the sdk locally before its release. Do we also need to change the way we build the sdk in order to do testing?

Reply:
SDK is released by SDK team, we just use it.
This version has been released at May: https://pypi.org/project/azure-mgmt-containerregistry/#history